### PR TITLE
issue #827 update the cron query to execute in batch mode

### PR DESCRIPTION
**Pull request by zqian**
Test Plan:

1. add MyLA tool to multiple courses
2. test the UDW query with "ENV_FILE=/secrets/env.hjson" in .env file
3. test the UDP query with "ENV_FILE=/secrets/env_udp.hjson" in .env file

### CODE REVIEW

File: dashboard/cron.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
 import logging
 from collections import namedtuple
+from operator import truediv
```
Comment (jonespm) [inline dashboard/cron.py]: It doesn't look like this truediv is being used.

Comment (zqian) [reply dashboard/cron.py]: also reported by Codacy. This line is removed.

**Review by jonespm — APPROVED** [review]
Ran both the UDP and UDW and looks like they run without error. 

The user validation is returning an expanded number for one of the queries now 

1.7700000000529132e+16
instead of
17700000000529132

I'm not sure what's changed with that. It looks fine though.

**Comment by zqian** [comment]
I saw that data type change too for course id field in validation script. Don't know why either. 

I will merge in the PR, and test the execution time length of cron job.

**Comment by ssciolla** [comment]
@zqian, @jonespm, probably this? https://re-thought.com/how-to-suppress-scientific-notation-in-pandas/

**Pull request merged**